app/src/organisms/GripperWizardFlows/MovePin.tsx

```diff
@@ -270,7 +277,7 @@ export const MovePin = (props: MovePinProps): JSX.Element | null => {
       proceedButtonText={buttonText}
       proceed={handleOnClick}
       proceedIsDisabled={maintenanceRunId == null}
-      back={goBack}
+      back={back}
```

Comment:
can this be refactored into this?

```suggestion
      back={movement !== MOVE_PIN_TO_FRONT_JAW ? goBack : undefined}
```

Reply:
yes— I originally used the same logic, but I wasn't sure if it was good practice to reference `undefined` explicitly. I'll update!

Reply:
I think it's a preference to reference it explicitly or not. I try to avoid it as well but In this case, since `back` can be 2 things, imo it is cleanest to just put the logic in a ternary.